### DISCUSSION TITLE
Is there a way to bind a mouse button to open a menu?

### DISCUSSION THREAD

**Discussion by CarlQLange** [Schneegans/Fly-Pie#217] [Fly-Pie Q&A] [answered]
I kind of _just_ want to use the mouse to open fly-pie, no keyboard involved at all. I've got a Logitech MX Master with too many buttons - is there a way for me to bind a mouse button (eg, mouse 5) to open fly-pie natively? Or do I need to do some hax to first bind that button to a keypress with Logiops or xbindkeys or something?

Awesome extension by the way, the amount of effort you've put in really shows through!

**Comment by Schneegans** [comment, marked as answer]
AFAIK, it's impossible for a GNOME Shell extension to bind something globally to a mouse button. Nevertheless, there are some third-party possibilities as you already pointed out. I can suggest these, for example:
* [Custom Hot Corners - Extended](https://extensions.gnome.org/extension/4167/custom-hot-corners-extended/): Open menus by moving your mouse to one corner of your screen!
* [Input Remapper](https://github.com/sezanzeb/input-remapper/): Bind menus to your additional mouse buttons!
* [xbindkeys](http://www.nongnu.org/xbindkeys/xbindkeys.html): X11-only alternative to Input Remapper.
* [Easystroke](https://github.com/thjaeger/easystroke/wiki): X11 only, use mouse gestures to open menus!

**Reply by CarlQLange** [reply]
That's great, thanks! Good to know it's a technical requirement. I ended up using `logiops` to bind a key to a mouse button and used that.
